### ISSUE TITLE
Eyepatch + Eyepatch Huds flipped don't work/look right

### ISSUE THREAD

**Issue by ps3moira** [space-wizards/space-station-14#29169]
## Description

Eyepatches don't look right if you flip them and there's no green-outline when you hover your cursor of their icon if they're flipped. 
**Reproduction**

Flip the eye patch.
**Screenshots**

![image](https://github.com/space-wizards/space-station-14/assets/113228053/9b99f873-685d-47d5-abde-7ada4e70115b)
My cursor is over the item but there's no green outline.
![image](https://github.com/space-wizards/space-station-14/assets/113228053/21dcaa5a-c28d-4aa4-8a74-acdc1ef467cd)

**Additional context**

From the looks of the PR Makes Eyepatches Flippable #26277 there was originally a proper working implementation but was told to change because duplicate of the same sprites are bad. This causes issues though as the way the sprite are flipped now with components makes it more broken.

**Comment by Cojoke-dot** [comment]
Yup, another issue is that the left and right sides of the head need to be swapped when flipped. The back of the head also needs a displacement like the front.
